service/routes.py

```diff
@@ -173,6 +173,30 @@ def get(self, shopcart_id):
         app.logger.info("Returning shopcart: %s", shopcart.id)
         return shopcart.serialize(), status.HTTP_200_OK
 
+    @api.doc("update_shopcarts")
+    @api.response(404, "Shopcart not found")
+    @api.response(400, "The posted Shopcart data was not valid")
+    @api.expect(shopcart_base_model)  # Updated the expect decorator
+    @api.marshal_with(shopcart_model)
+    def put(self, shopcart_id):
+        """
+        Update a Shopcart
+
+        This endpoint will update a Shopcart based on the body that is posted
+        """
```

Comment:
`check_content_type(DEFAULT_CONTENT_TYPE)` at [line:270](https://github.com/CSCI-GA-2820-SU23-001/shopcarts/pull/143/files#diff-5713d882378afffa24127e3725abe088589f9e300b01fa079a5a80179d2f6c1eR270) in the original implementation is missing, please help to add it back!

Reply:
done